### DISCUSSION TITLE
About abbreviation style

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#6905] [Q&A] [answered]
In abbreviations, how to change the mouse hover style to the official document? I think the default effect is not very good-looking. Is this a donation-only feature?
As shown below
![8eb84ea52782bca269df93fb279fd8de](https://github.com/squidfunk/mkdocs-material/assets/109416673/66442b15-10d7-48ff-81bf-ab30903891a3)
![72f6a44141305b7882cddca6adeb8767](https://github.com/squidfunk/mkdocs-material/assets/109416673/1f09ea9c-f9b8-45f6-b61b-e94189d261aa)

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @W1ndys,
until recently this was an Insiders only feature, but now it's in the community version too, since version 9.5.0:
- https://squidfunk.github.io/mkdocs-material/insiders/#14000-goats-horn
- https://squidfunk.github.io/mkdocs-material/reference/tooltips/#improved-tooltips

Though after the adoption into the community version, some issues revealed themselves concerning positioning, so please keep that in mind and wait for a fix :v::
- https://github.com/squidfunk/mkdocs-material/issues/6488

**Reply by w1ndys** [reply]
I see, thank you very much!
